tests/yast2_gui/yast2_control_center.pm

```diff
@@ -8,7 +8,7 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# Summary: Extend YaST2 UI test yast2-control-center
+# Summary: Make YaST2 UI test yast2-control-center more reliable and avoid timeout error
```

Comment:
no, that does not help. "Extend UI test" is also not a good summary of what the test module does